py/Configuration.py

```diff
@@ -28,6 +28,9 @@ def initialize(load_interface=True):
 
         Configuration.verbose = 0 # Verbosity level.
 
+	Configuration.scan_time = 0 # Scan time
```

Comment:
Indent/whitespace on this line seems off (one space short)... looks like it might be a tab `\t` character! 

I thought Python threw a fit if the file mixes tabs with spaces...